src/main/java/Calculator.java

```diff
@@ -0,0 +1,73 @@
+// It's 1949 and you are tasked to lead programmer's team in order to implement a brand new calculator. The calculator is very simple though: it can add (+), multiple(x), divide(%) and take(-). It does not and will not support any brackets or memory. It will only ever support whole numbers.
+// Calculator should have only 1 function- int calculate(string expression)
+
+public class Calculator {
+
+    private char[] defaultOperators = {'+', '-', 'x', '%', '/', '!'};
+    private char[] charArray;
```

Comment:
Should it really be in a class scope? I would say it makes more sense to have it in the method itself, because this is now like a temporary variable and it could be messed up if we interupt a calculation with a new one